sdcm/cluster.py

```diff
@@ -1330,6 +1336,15 @@ def _read_system_log_and_publish_events(self,
                             one_line_backtrace.append(trace_line)
 
                 if index not in self._system_log_errors_index or start_from_beginning:
+                    ''' for each line, if it matches a continuous event pattern, 
+                    create a new event and call the appropriate func '''
+                    db_event_pattern_func_map = get_pattern_to_event_to_func_mapping()
+                    for item in db_event_pattern_func_map:
```

Comment:
we need to have the same ID, look in the registry that you created in one of the previous PRs)

Reply:
Actually the concept of "same id" is problematic here, since we're looking at log lines and they won't help us with the question "which event id should this log line be connected to?". E.g. if we have more than one ContinuousEvent of a given type and we encounter a "End" line for that type of event, which event should be ended? My currently implemented solution is to add a `node` attribute to ContinuousEvent and leverage the ContinousEventRegistry. When we encounter a "end" line in the log, we:
1. Filter registry events by node.
2. Filter (1) by event type.
3. Filter (2) by period (i.e. only get begun events)
4. Check if we got only 1 event and warn the user if we got more than 1.